src/components/enterprise-redirects/tests/EnterpriseLearnerFirstVisitRedirect.test.jsx

```diff
@@ -19,14 +21,59 @@ jest.mock('react-router-dom', () => ({
   }),
 }));
 
+const defaultUserSubsidyState = {
+  redeemableLearnerCreditPolicies: [{
+    learnerContentAssignments: {
+      state: 'allocated',
+    },
+  },
+  {
+    learnerContentAssignments: {
+      state: 'accepted',
+    },
+  }],
+};
+
+const EnterpriseLearnerFirstVisitRedirectWrapper = ({
+  initialUserSubsidyState = defaultUserSubsidyState,
+}) => (
+  <IntlProvider locale="en">
```

Comment:
[curious] Is `<IntlProvider>` required by this component? If not, probably don't need to include it.

Reply:
Yep you're right, I always throw it in just incase the test errors on the locale. Removed now 👍🏽